pkg/ignition/ignition.go

```diff
@@ -6,8 +6,8 @@ import (
 	"sort"
 	"strings"
 
-	ignCfg "github.com/coreos/ignition/v2/config/v3_2"
-	ignCfgTypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	ignCfg "github.com/coreos/ignition/v2/config/v3_4"
+	ignCfgTypes "github.com/coreos/ignition/v2/config/v3_4/types"
```

Comment:
I don't think this is needed. [xref](https://github.com/openshift/windows-machine-config-operator/pull/1728#issue-1850346835)

cc @jkyros   Thoughts?

Reply:
Assuming upgrade order is such that the WMCO gets upgraded _after_ the MCO, there is potential at least for the 4.13 WMCO to receive a `3.4.0` version from the 4.14 MCO during an upgrade, so having the 4.13 MCO understand `3.4.0` seems like a good idea?